concordia/admin/views.py

```diff
@@ -225,13 +225,15 @@ def admin_bulk_import_review(request):
                 all_urls.append(urls)
                 for i, val in enumerate(all_urls):
                     return_result = fetch_all_urls(val)
-                    messages.info(request, f"Asset Count:{return_result[0]}")
+                    for res in return_result[0]:
+                        messages.info(request, f"{res}")
```

Comment:
This will create one message per character in the returned string. I think this block should be `message, count = fetch_all_urls(val)`.

Reply:
I want to display one message at a time from the thread workers when they return, so it shows the asset count per url, and the asset count and the total at the end.